scripts/lib/mkcloud-libvirt.sh

```diff
@@ -215,8 +34,9 @@ function libvirt_start_daemon()
 
 function libvirt_setupadmin()
 {
-    libvirt_onhost_create_adminnode_config
-    libvirt_onhost_create_admin_network_config
+    onhost_local_repository_mount
+    python ${mkcloud_lib_dir}/libvirt/admin-config.py $cloud $admin_node_memory $adminvcpus $emulator $admin_node_disk "$local_repository_mount"
```

Comment:
Yes, these scripts should definitely be executable.  +1 for @aplanas's suggestion here too.